mr_module/TimeMachine/TimeMachineController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace MrModule\TimeMachine;
+
+use Mr\Http\Controllers\Controller;
+use MrModule\TimeMachine\TimeMachine;
+
+class TimeMachineController extends Controller
+{
+    public function index() {
+        $results = TimeMachine::all();
+        return response()->json($results);
+    }
+
+    protected function show($id) {
+        $result = TimeMachine::findOrFail($id);
+        return response()->json($result);
+    }
+
+    protected function destroy($id) {
+        $result = TimeMachine::findOrFail($id);
+        $result->delete();
+
+        return response()->setStatusCode(204);
+    }
+
+    public function stats() {
+        $lastDay = new \DateTime();
+        $lastDay->sub(new \DateInterval('P1D'));
+
+        $lastWeek = new \DateTime();
+        $lastWeek->sub(new \DateInterval('P1W'));
+
+        $todayCount = TimeMachine::where('last_success', '>', $lastDay)->count();
```

Comment:
This is how i am doing stats at the moment, but we CAN use a CASE() statement if it gets too slow.